lib/experimental/class-gutenberg-rest-global-styles-revisions-controller.php

```diff
@@ -265,10 +265,13 @@ public function get_item_permissions_check( $request ) {
 			return $post;
 		}
 
+		/*
+		 * The same check as WP_REST_Global_Styles_Controller->get_item_permissions_check.
+		 */
 		if ( ! current_user_can( 'read_post', $post->ID ) ) {
 			return new WP_Error(
 				'rest_cannot_view',
-				__( 'Sorry, you are not allowed to view revisions for this global style.', 'gutenberg' ),
+				__( 'Sorry, you are not allowed to view this global style.', 'gutenberg' ),
```

Comment:
Just double-checking, the only intended change in this PR is the wording on this line + adding tests?

In [the other PR](https://github.com/WordPress/gutenberg/pull/50089/files#r1183152555) it looks like a different check was being used. In this case, the `current_user_can( 'read_post', $post->ID )` check appears to be consistent with `WP_REST_Global_Styles_Controller` 👍

Reply:
Sorry 🤦 

The intention was to ensure the check was exactly the same as WP_REST_Global_Styles_Controller and I copied the translation over as well. 

That means the only purpose of this PR is to add 1 x comment and add a test. 

Thanks @andrewserong

Reply:
All good, thanks for clarifying! 🙂